sky_provider/example_dags/sky_aws_credentials.py

```diff
@@ -0,0 +1,29 @@
+from airflow.decorators import dag
```

Comment:
Should the `example_dags` be in the package itself or at the root level

Reply:
I think we can exclude it from the package.

Reply:
Moved